dispatcher/MongoConnect.py

```diff
@@ -362,22 +362,33 @@ def GetNextRunNumber(self):
             return 0
         return list(cursor)[0]['number']+1
 
-    def SetStopTime(self, number, detector, force):
+    def SetStopTime(self, number, detectors, force):
         '''
         Sets the 'end' field of the run doc to the time when the STOP command was ack'd
         '''
-        self.log.info("Updating run %i with end time"%number)
+        self.log.info("Updating run %i with end time (%s)" %(number, detectors))
         try:
             time.sleep(2) # this number depends on the delay between CC and reader stop
             endtime = self.GetAckTime(detector, 'stop')
```

Comment:
this should say detectors

Reply:
It probably should, but how many people ever read the dispatcher logs?

Reply:
No I mean `detector` is not defined in `endtime = self.GetAckTime(detector, 'stop')`

Reply:
Ah, you mean a typo. I was looking at the highlighted change lines :rofl: